compiler/qsc_data_structures/src/namespaces/tests.rs

```diff
@@ -0,0 +1,365 @@
+use expect_test::expect;
+
+use super::*;
+
+#[allow(clippy::too_many_lines)]
+#[test]
+fn test_tree_construction() {
+    let mut root = NamespaceTreeRoot::default();
```

Comment:
It might be better for testing purposes if there was a way to construct a default tree root without the prelude.

Reply:
It's not absolutely necessary, so I'll go ahead an resolve this comment.